src/gtk/dataview.cpp

```diff
@@ -1152,9 +1152,33 @@ extern "C" {
 #define GTK_TYPE_WX_CELL_EDITOR_BIN               (gtk_wx_cell_editor_bin_get_type ())
 #define GTK_WX_CELL_EDITOR_BIN(obj)               (G_TYPE_CHECK_INSTANCE_CAST ((obj), GTK_TYPE_WX_CELL_EDITOR_BIN, GtkWxCellEditorBin))
 
+// In GTK+ < 3.8 GtkBin can't be used as a widget parent without defining our
```

Comment:
"widget parent" is a bit confusing here. Maybe something like "base type" or "parent class".